app/models/thredded/null_user.rb

```diff
@@ -34,6 +40,8 @@ def thredded_anonymous?
       true
     end
 
+    alias_method :anonymous?, :thredded_anonymous?
```

Comment:
In the real User class, this is also `thredded_anonymous?`: https://github.com/thredded/thredded/blob/8dedc824487063e2cc989c29af0b231c9c9485cd/app/models/thredded/user_extender.rb#L23